src/daily-quest/daily-quest.forms.ts

```diff
@@ -39,10 +39,32 @@ export const DailyQuestForm = z
       path: ["frequency"],
     }
   );
-export type DailyQuestFormType = z.infer<typeof DailyQuestForm>;
-
-export const CreateDailyQuestForm = DailyQuestForm;
 export type CreateDailyQuestFormType = z.infer<typeof CreateDailyQuestForm>;
 
-export const UpdateDailyQuestForm = DailyQuestForm;
+export const UpdateDailyQuestForm = z
```

Comment:
I didn't find an easy way to implement Forms inheritance with Zod objects, so did this in straight forward way with a bit of code duplication.